libs/portals/my-pages/health/src/screens/Waitlists/Waitlists.graphql

```diff
@@ -0,0 +1,12 @@
+query GetWaitlists($locale: String) {
+  healthDirectorateWaitlists(locale: $locale) {
+    waitlists {
+      id
+      name
+      waitBegan
+      lastUpdated
+      organization
+      status
+    }
+  }
+}
```

Comment:
Maybe add a new endpoint, `healthDirectorateWaitlist` which accepts an `id`. Then the query only returns 1 result instead of a list which needs to be filtered.